src/routes/dashboard/(draft)/students/+page.server.ts

```diff
@@ -118,7 +118,9 @@ export const actions = {
     await dispatch.dispatchRoundSubmittedNotification(lab, name, draftId, deferredNotifications[0]);
 
     db.logger.info('student rankings submitted');
-    for (const round of deferredNotifications)
-      await dispatch.dispatchDraftRoundStartNotification(draftId, round);
+
+    await dispatch.bulkDispatchDraftRoundStartNotification(deferredNotifications.map(round => {
+      return { draftId, draftRound: round };
```

Comment:
We _technically_ need a `satisfies` clause here, but since it's inlined at the call site of the dispatch, the type checker should flag it in case the literal didn't have the correct structure.

Nevertheless, I recommend that we annotate this correctly for the sake of documentation.